### ISSUE TITLE
Provide sub sort option

### ISSUE THREAD

**Issue by juninhocruzg3** [neilsoult/typescript-imports-sort#3]
This extension could provide option for sub sort, so that I'll can configure for root sort by groups (package, uplevel, downlevel) and for second sort by options (importName, path or filename).

**Comment by neilsoult** [comment]
could you provide me with an example of imports and how you would like them sorted by subgroups?

**Comment by juninhocruzg3** [comment]
Something like this:
```
// Sorted by groups, and sub-sorted by filename (last folder on the path)

// Package group
import * as LRU from 'lru-cache' // L
import * as mongoose from "mongoose" // M
import * as restify from "restify" // R
import { Sequelize } from 'sequelize-typescript' // S

// Uplevel group
import { checkAuthorization } from '../security/authorization' // A
import { Company } from '../companies/companies.model' // C
import { environment } from '../common/environment' // E
import { DefaultResponse, Router } from "../common/router" // R
import { parserToken, Token } from "../security/token.parser" // T

// Downlevel group
import { alter, DBUpdate } from './scripts/alter' // A
import { handlerError } from './error.handler' // E
import { populate } from './scripts/populate' // P
```

I ask for "sub-sort" and "sort by filename"

**Comment by neilsoult** [comment]
Released in v1.11.0
